components/AppointmentCard.js

```diff
@@ -0,0 +1,70 @@
+import styled from "styled-components";
+import { AiOutlineClockCircle } from "react-icons/ai";
+
+export default function Card({ date, name, text, clock, green }) {
+  return (
+    <CardWrapper>
+      <Green>{green}</Green>
+      <Text>{text}</Text>
+      <Name>{name}</Name>
+      <AiOutlineClockCircle size="1rem" color="grey" />
+      <Date>{date}</Date>
+      <Clock>{clock}</Clock>
+    </CardWrapper>
+  );
+}
+
+const CardWrapper = styled.li`
+  display: grid;
+  margin: 2px 5px 2px 5px;
+  grid-template-columns: 1.8fr 2fr 5fr;
+  grid-template-rows: 1fr 1fr;
+  /* grid-template-areas:
+    "a b c"
+    "d e f"; */
+  align-content: space-between;
+  padding: 10px;
+  background-color: white;
+  border-bottom: 1.6px solid #dadada;
+`;
+
+const Name = styled.div`
```

Comment:
if you are displaying text , use a HTML Tag for Text, `p` or `span` maybe? check also for the following items.